src/content/docs/infrastructure/host-integrations/host-integrations-list/lightptd-integration.mdx

```diff
@@ -0,0 +1,152 @@
+
+<img
+  title="Lighttpd dashboard"
+  alt="A screenshot depicting the Lighttpd dashboard"
+  src={infrastructureLighttptdDashboard}
+/>
+
+<figcaption>
+After setting up the integration with New Relic, see your data in dashboards like these, right out of the box.
+</figcaption>
+
+## Install the infrastructure agent [#infra]
+
+To get data into New Relic, install our infrastructure agent. Our infrastructure agent collects and ingests data so you can keep track of your app's performance. The version should be 1.10.7 or later to support NRI-Flex integration.
+
+You can install the infrastructure agent two different ways:
+
+* Our [guided install](https://one.newrelic.com/nr1-core?state=4f81feab-35f7-e97e-9903-52510f8542bd) is a CLI tool that inspects your system and installs the infrastructure agent alongside the application monitoring agent that best works for your system. To learn more about how our guided install works, check out our [Guided install overview](/docs/infrastructure/host-integrations/installation/new-relic-guided-install-overview).
+* If you'd rather install our infrastructure agent manually, you can follow a tutorial for manual installation for [Linux](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux), [Windows](/docs/infrastructure/install-infrastructure-agent/windows-installation/install-infrastructure-monitoring-agent-windows/), or [macOS](/docs/infrastructure/install-infrastructure-agent/macos-installation/install-infrastructure-monitoring-agent-macos/).
+
+## Integrate Lighttpd with New Relic [#integrate]
+
+Once installed, the infrastructure agent can ingest data from your app and send it to New Relic, but you still need to integrate – or establish a line of communication – between the agent and your app. Once integrated, you can get started with an out of the box monitoring solution for your Lighttpd app. 
+
+<Steps>
+<Step>
+### Manually configure the lighttpd.conf file
+
+To export metrics, please use the following steps on `127.0.0.1:9880/server-status?format=plain`
+
+1. Go to lighttpd config file path
+```
+cd /etc/lighttpd/
+```
+2. Open `lighttpd.conf` file to edit
+```
+sudo nano lighttpd.conf
+```
+3. Add `mod_status` on your Lighttpd servers.
+4. Add `mod_status` inside `server.modules` object.
+5. Add status.status-url = `/server-status` after server.modules
+6. Update `server.document-root` with the lighttpd file path `/var/www/html/lighttpd-webpage`
+7. Update `server.port` on which you are running lighttpd server.port = 9880
+8. Run `127.0.0.1:9880/server-status?auto` or  `127.0.0.1:9880/server-status?format=plain` on the browser to check server related metrics.
```

Comment:
after the 8th step, please add the below text and configuration.

Here is the sample configuration file which looks like after following all the above steps.

```
server.modules = (
    "mod_1indexfile",
    "mod_access”,
    "mod_alias",
    "mod_redirect”,
    "mod_status"
)

server.document-root	= "/var/www/html/1lighttpd-webpage"
server.upload-dirs	  = ( "/var/cache/lighttpd/uploads" )
server.errorlog		    = "/var/log/lighttpd/error.log"
server.pid-file		    = "/run/lighttpd.pid"
server.username		    = "www-data"
server.groupname	    = "www-data"
server.port		        = 9880
status.status-url	    = "/server-status”

```

Reply:
Done.